UiPharmecy/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
[Same](https://github.com/AhsanBracu/Image-To-Pdf/pull/1/files#diff-a084b794bc0759e7a6b77810e01874f2R42)